chef/cookbooks/provisioner/recipes/base.rb

```diff
@@ -351,7 +351,9 @@
 protocol = crowbar_node["crowbar"]["apache"]["ssl"] ? "https" : "http"
 server = "#{protocol}://#{address}"
 password = crowbar_node["crowbar"]["users"]["crowbar"]["password"]
-verify_ssl = !crowbar_node["crowbar"]["apache"]["insecure"]
+verify_ssl = !crowbar_node["crowbar"]["apache"]["insecure"] ||
+  crowbar_node["crowbar"]["apache"]["generate_certs"]
```

Comment:
Don't you mean `&& !crowbar_node["crowbar"]["apache"]["generate_certs"]` ?

Reply:
Thanks @jsuchome I missed that.